packages/sui-js/src/string/index.js

```diff
@@ -1,9 +1,13 @@
-import {parse} from 'qs'
+import {parse, stringify} from 'qs'
 
 export const parseQueryString = query => parse(query, {ignoreQueryPrefix: true})
+export const arrayToCommaQueryString = query =>
+  unescape(stringify(query, {arrayFormat: 'comma'}))
```

Comment:
unescape 'fn' is deprecated
why not import `stringify` and send the options from outside? I think that this is unnecessary

Reply:
changed it to `decodeUriComponent()`